README.md

```diff
@@ -155,6 +155,20 @@ install-nix-action's own `extra_nix_config` input:
       substituters = https://hydra.iohk.io https://cache.nixos.org/
 ```
 
+### How do I use `nix develop`?
```

Comment:
I would prefer for the title to focus on what this accomplishes vs what the underlying command is.

For example, "How do I configure steps to use my flake's development environment?"

The next few sentences can then expand on that by saying that you can configure [`jobs.<job_id>.steps[*.]shell`](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#jobsjob_idstepsshell) (with a link) to use `nix develop`.